app/models/bgs_dependents/base.rb

```diff
@@ -2,6 +2,14 @@
 
 module BGSDependents
   class Base < Common::Base
+    include ActiveModel::Validations
+
+    def dependent_address(dependents_application, lives_with_vet, alt_address)
```

Comment:
Prefer named [keyword args](https://thoughtbot.com/blog/ruby-2-keyword-arguments). Being more explicit is better than being less explicit. Kwargs will provide better failure messages.

Reply:
Makes sense, thanks.  Done.